Specs/NumberWithUnit/Dutch/CurrencyModel.json

```diff
@@ -0,0 +1,276 @@
+[
+  {
+    "Input": "Hey, het gaat $4,25 kosten en 32 is het aantal dat ik wil afnemen!",
+    "Results": [
+      {
+        "Text": "$4,25",
+        "TypeName": "currency",
+        "Resolution": {
+          "unit": "Dollar",
+          "value": "4,25"
+        }
+      }
+    ],
+    "NotSupportedByDesign": "javascript,python,java"
+  },
+  {
+    "Input": "Het bedrag is 5 euro 50",
+    "Results": [
+      {
+        "Text": "5 euro 50",
+        "TypeName": "currency",
+        "Resolution": {
+          "unit": "Euro",
```

Comment:
Shouldn't this have an `IsoCurrency`?

Reply:
What do you mean? I meant that the property ‘isoCurrency’ is present for single unit results, however not in the MergedUnit result.
5 euro vs 5 euro 50